api-flask/app/nginx.conf

```diff
@@ -0,0 +1,59 @@
+# Nginx config.
+
+# Allow large URI.
+client_header_buffer_size 64k;
+large_client_header_buffers 4 64k;
+
+# Avoid timeouts for large files.
+proxy_connect_timeout       2000;
+proxy_send_timeout          2000;
+proxy_read_timeout          2000;
+send_timeout                2000;
+
+uwsgi_read_timeout          2000;
+uwsgi_send_timeout          2000;
+
+# Avoid 413s for large POST bodies.
+client_max_body_size        30m;
+
+# Http server settings.
+server {
+
+    listen 80;
```

Comment:
Probably want to [use nginx to redirect http -> https](https://serversforhackers.com/c/redirect-http-to-https-nginx) and listen for https connections.

Reply:
Sorry the ngninx confi wasnt supposed to be part of the PR. I just wanted to show how you would go about adding one. In case we need to play with it for https